### ISSUE TITLE
pip_import rule with invalid requirements.txt file does not throw an error

### ISSUE THREAD

**Issue by macrael** [bazelbuild/rules_python#19]
With a directory layout like so: 

/WORKSPACE
/common
/common/requirements.txt

None of the following attempts at the requirements string in pip_import worked:

"//common/requirements.txt"
"//common:requirements.txt"
"common/requirements.txt"

And the only failure reported was the `load("@gs_pip_deps//:requirements.bzl", "pip_install")` invocation. `pip_import` itself should report an error instead of silently succeeding.

To fix this I had to add a BUILD file to common, set it to visibility public, and use "//common:requirements.txt"

**Comment by thundergolfer** [comment]
This appears quite similar to the recent issue https://github.com/bazelbuild/rules_python/issues/299

See in that issue my explanation that `BUILD` files are required in Bazel when referring to files as targets.

When I try `pip_import` without a `BUILD` locally (using latest code) I get an immediate error: 

```
INFO: Call stack for the definition of repository 'my_deps' which is a pip_import (rule definition at /private/var/tmp/_bazel_jbelotti/a8a1f81a1422a72c79c6c81641a8c816/external/rules_python/python/pip.bzl:51:14):
 - /Users/jbelotti/work/foo/WORKSPACE:96:1
ERROR: An error occurred during the fetch of repository 'my_deps':
   Unable to load package for //:requirements.txt: BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - /Users/jbelotti/work/foo
```

I will close this issue now. Let me know if it needs to be reopened.
